vllm/core/scheduler.py

```diff
@@ -1755,6 +1935,8 @@ def _get_num_new_uncached_and_cached_tokens(
                 budget,
                 self._get_prompt_limit(seq_group),
                 num_uncached_new_tokens,
+                self.partial_prefill_budget_lookup_list,
+                partial_prefill_metadata,
```

Comment:
nit: I think `_chunk_new_tokens_to_schedule` should not be static if this many of arguments are instance attributes

But doesn't have to block this PR